actors/miner/src/deadlines.rs

```diff
@@ -128,6 +128,26 @@ pub fn deadline_available_for_compaction(
         )
 }
 
+// the distance between from_deadline and to_deadline clockwise in deadline unit.
+fn deadline_distance(policy: &Policy, from_deadline: u64, to_deadline: u64) -> u64 {
+    if to_deadline > from_deadline {
+        to_deadline - from_deadline
+    } else {
+        policy.wpost_period_deadlines - from_deadline + to_deadline
```

Comment:
```suggestion
    if to_deadline >= from_deadline {
        to_deadline - from_deadline
    } else {
        policy.wpost_period_deadlines - from_deadline + to_deadline
```

The distance should be 0~47.  
And, this change makes the function deadline_available_for_move more reasonable. It will automatically return false if currDeadline=fromDeadline

Reply:
Good catch, fixed!